File: unity/Assets/Scripts/Multiplayer/Rooms/RoomKickRequest.cs
using System;
using UnityEngine;

namespace Michitai.Multiplayer.Rooms
{
    /// <summary>
    /// Request data for kicking a player from a game room in Unity.
    /// </summary>
    [Serializable]
    public class RoomKickRequest
    {
        /// <summary>
        /// The ID of the player to kick.
        /// </summary>
        public int player_id;
    }
}

File: java/src/main/java/com/michitai/multiplayer/rooms/RoomKickRequest.java
package com.michitai.multiplayer.rooms;

import com.fasterxml.jackson.annotation.JsonProperty;

/**
 * Request for kicking a player from a room.
 */
public class RoomKickRequest {
    @JsonProperty("player_id")
    private int playerId;

    public RoomKickRequest(int playerId) {
        this.playerId = playerId;
    }

    public int getPlayerId() {
        return playerId;
    }

    public void setPlayerId(int playerId) {
        this.playerId = playerId;
    }
}
